plugins/scaffolder-backend/src/scaffolder/stages/publish/github.ts

```diff
@@ -59,9 +61,22 @@ export class GithubPublisher implements PublisherBase {
   }: PublisherOptions): Promise<PublisherResult> {
     const { owner, name } = parseGitUrl(values.storePath);
 
+    const token =
```

Comment:
Bit nicer to deconstruct the response instead:

```ts
const { token } = await this.config.credentialsProvider.getCredentials({ url: values.storePath })
```

Also prolly better to not default the token to `''` and instead just fail early with an error saying that we couldn't acquire credentials to publish to the given store path.

Reply:
I added a catch for when the token is undefined and returned:
`return { remoteUrl: '', catalogInfoUrl: undefined };`
I think we could add a logger.error message or something like that before returning. What do you think? What should the error say? "Unable to adquire credentials for publisher."

Reply:
or maybe:
`logger.error(`Unable to acquire credentials for ${owner}/${name}`);`